src/site/_includes/components/header/header.js

```diff
@@ -0,0 +1,28 @@
+const hdr = (function() {
```

Comment:
Waarom is dit een IIFE? Als je deze file gewoon zou importeren in app.js zou de code erin toch gewoon moeten werken? Of ben je bang voor naming conflicts met andere componenten? Want dan snap ik 'm opzich wel.

Anyway, `hdr` kan natuurlijk gewoon beschrijvender, en anders kan je simpelweg 't volgende doen I guess:

```js
export default (() => {
  // Function dingen
})()
```

En dan een arrow function ervan maken zoals bovenstaand aangezien we dat bij bijvoorbeeld zo'n filter ook direct doen met `module.exports = () => {}`

Reply:
Als we dit dan consistent houden voor elk component zoals bovenstaand met meteen die `export default` vind ik dit wel een erg nette oplossing om zo de js van elk component bij elkaar te houden.

Reply:
Ja is voornamelijk op naming conflicts gericht. Maar idd export default (() => {})() zou mooier zijn.